src/app/Signup/page.module.css

```diff
@@ -0,0 +1,134 @@
+.SignUpPageContainer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 100vh;
+  width: 100vw;
+}
+
+.EmailContainer {
+  align-items: center;
+  margin-bottom: 1rem;
+}
+.PasswordContainer {
+  align-items: center;
+  margin-bottom: 1rem;
+}
+.ConfirmPasswordContainer {
+  align-items: center;
+  margin-bottom: 2rem;
+}
+.TextBoxLabel {
+  margin-bottom: 0.2rem;
+  display: flex;
+}
+.span {
+  display: flex;
+  justify-content: center;
+}
+
+.CreateAccount {
+  margin: 3rem;
+  font-size: 3rem;
+  margin-bottom: 3rem;
+  margin-top: 2rem;
+}
+.lgnbttncontainer,
+.signbttncontainer {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+@media (min-width: 300px) and (max-width: 380px) {
+  .SignUpPageContainer {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    height: 100vh;
+  }
```

Comment:
delete, media queries remember what is defined outside of it and carries over. think of it like the previous CSS classes that are not in media queries are global variables.